src/components/organisms/AppRouter/AppRouter.jsx

```diff
@@ -60,6 +62,12 @@ const AppRouter = () => {
         <Route path="/admin/venue/creation" component={VenueWizard} />
         <Route path="/admin/venue/edit/:venueId" component={VenueWizard} />
 
+        <Route path="/admin_v2/venue/:venueId?" component={AdminVenueView} />
```

Comment:
if we end up creating functions for the routes in `utils/url` (like I've suggested in other places)  we can use it here as well, if we add a default value for the parameter there 
```ts
export const adminV3VenueUrl = (venueId : string = ':venudId?') => `/admin_v2/venue/${venueId}`
```

Reply:
I think react router may already have some helper functions designed for constructing urls in an efficient way. Could be worth checking those out too.

Reply:
Less relevant for this route, but for the others, I’m curious why we need to use absolute paths everywhere, rather than relative paths when navigating around the subsections of the admin panel. Relative would make it simpler/less rigid IMO

Reply:
@0xdevalias @sunny-viktoryia 
> if we end up creating functions for the routes in utils/url (like I've suggested in other places) we can use it here as well, if we add a default value for the parameter there
> ```export const adminV3VenueUrl = (venueId : string = ':venudId?') => '/admin_v2/venue/${venueId}'```

This is not going to work because some of our screens rely on `venueId` and if it exists. In this case it will always exist by falling back to `:venueId`

What I've done before is have something like this for the routes:
```tsx
const SCREENS = {
  ADMIN_VENUE: {
    path: '/admin-v3/venue/:venueId',
    url: (venueId: string) => `/admin-v3/venue/${venueId}`,
    component: AdminVenue
  },
  ADMIN_SETTINGS: {
    path: '/admin-v3/advanced-settings/:venueId',
    url: (venueId: string) => `/admin-v3/advanced-settings/${venueId}`,
    component: AdminAdvancedSettings
  }
  ....
}



Usage: 

<Route path={SCREENS.ADMIN_VENUE.path} component={SCREENS.ADMIN_VENUE.component} /> 

and 

<Link to={SCREENS.ADMIN_VENUE.url(venueId)}  />
```

The only problem is that this introduces small duplication because paths without params are going to be exactly like the urls.

Ex: 
```tsx
    path: '/home/entrance',
    url: '/home/entrance',
```

Reply:
https://reactrouter.com/web/api/generatePath

Reply:
You can handle an optional venueId in the function with a ternary?

const foo = (venueId?: string) => venueId ? ‘/foo/${venueId}’ : ‘/foo’

Reply:
I just used `generatePath` for the functions: https://github.com/sparkletown/sparkle/pull/1511/commits/2afccca403af90d38dca5a81a91041709b2afac3